### ISSUE TITLE
i see major problems

### ISSUE THREAD

**Issue by linustouchtips** [Sxmurai/inferno#50]
why is this client so chinese lol

**Comment by Sxmurai** [comment]
its not exactly ready, as ive said in the readme. i havent taken a huge amount of time to prefect old shitty code, which is a todo rn. im mostly focused on getting the features id atleast like, and then improving later.

**Comment by linustouchtips** [comment]
do you have a discord

**Comment by Sxmurai** [comment]
i do, i havent exactly released it yet. unless you were talking about my @

**Comment by linustouchtips** [comment]
add me: linus touch tips#7672

**Comment by Sxmurai** [comment]
sent

**Comment by HausemasterIssue** [comment]
this client will be amazing lol, even if the src is chinese, functionality and how good the modules are is what's important.

**Comment by Sxmurai** [comment]
nah, im taking his advice and fixing shit i shouldve done from the start

**Comment by linustouchtips** [comment]
if the src is chinese, then by proxy the functionality and modules are chinese

**Comment by HausemasterIssue** [comment]
yeah, i guess
